src/components/Categories.js

```diff
@@ -1,8 +1,5 @@
-/* eslint-disable */
-import React from "react";
+import React from 'react';
 
-const Categories = () => {
-	return <button>Check Status</button>;
-};
+const Categories = () => <button type="button">Check Status</button>;
```

Comment:
- [ ] When the check status button is clicked, it is expected that you return a string showing 'under construction'. This doesn't happen yet. Please implement this requirement 🙏

Reply:
Hi @codepantha 
I appreciate your feedback. However, dispatching actions are not part of the requirements, for that, I'll have to make use of the provider, store, and redux hooks (useselector() and usedispatch()).

Reply:
@LuisSalas94, Sorry I couldn't respond earlier.  Adding and Removing books need to work as stipulated in the project requirements.